feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/aggregate_all_not_forwarding_viable_test.go

```diff
@@ -0,0 +1,1025 @@
+	bgp := niProto.GetOrCreateBgp()
+
+	global := bgp.GetOrCreateGlobal()
+	global.RouterId = ygot.String(dutLoopback.IPv4)
+	global.As = ygot.Uint32(asn)
+	global.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST).Enabled = ygot.Bool(true)
+	global.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV6_UNICAST).Enabled = ygot.Bool(true)
+	pgName := "BGP-PEER-GROUP1"
+	pg := bgp.GetOrCreatePeerGroup(pgName)
+	pg.PeerAs = ygot.Uint32(asn)
+	if deviations.RoutePolicyUnderAFIUnsupported(dut) {
+		rpl := pg.GetOrCreateApplyPolicy()
+		rpl.SetExportPolicy([]string{acceptRoutePolicy})
+		rpl.SetImportPolicy([]string{acceptRoutePolicy})
+	} else {
+		af4 := pg.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST)
+		af4.Enabled = ygot.Bool(true)
+		rpl := af4.GetOrCreateApplyPolicy()
+		rpl.SetExportPolicy([]string{acceptRoutePolicy})
+		rpl.SetImportPolicy([]string{acceptRoutePolicy})
+
+		af6 := pg.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV6_UNICAST)
+		af6.Enabled = ygot.Bool(true)
+		rpl = af6.GetOrCreateApplyPolicy()
+		rpl.SetExportPolicy([]string{acceptRoutePolicy})
+		rpl.SetImportPolicy([]string{acceptRoutePolicy})
+	}
+
+	for _, a := range []*aggPortData{agg1, agg2, agg3} {
+		bgpNbrV4 := bgp.GetOrCreateNeighbor(a.ateIPv4)
+		bgpNbrV4.PeerGroup = ygot.String(pgName)
+		bgpNbrV4.PeerAs = ygot.Uint32(asn)
+		bgpNbrV4.Enabled = ygot.Bool(true)
+		bgpNbrV4T := bgpNbrV4.GetOrCreateTransport()
+
+		bgpNbrV4T.LocalAddress = ygot.String(a.dutIPv4)
+		af4 := bgpNbrV4.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST)
+		af4.Enabled = ygot.Bool(true)
+
+		af6 := bgpNbrV4.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV6_UNICAST)
+		af6.Enabled = ygot.Bool(false)
+
+		bgpNbrV6 := bgp.GetOrCreateNeighbor(a.ateIPv6)
+		bgpNbrV6.PeerGroup = ygot.String(pgName)
+		bgpNbrV6.PeerAs = ygot.Uint32(asn)
+		bgpNbrV6.Enabled = ygot.Bool(true)
+		bgpNbrV6T := bgpNbrV6.GetOrCreateTransport()
+
+		bgpNbrV6T.LocalAddress = ygot.String(a.dutIPv6)
+		af4 = bgpNbrV6.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST)
+		af4.Enabled = ygot.Bool(false)
+		af6 = bgpNbrV6.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV6_UNICAST)
+		af6.Enabled = ygot.Bool(true)
+	}
+
+	gnmi.Replace(t, dut, gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut)).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP").Config(), niProto)
+}
+
+// configureRoutingPolicy configure routing policy on DUT
+func configureRoutingPolicy(t *testing.T, dut *ondatra.DUTDevice) {
+	t.Helper()
+	d := &oc.Root{}
+	rp := d.GetOrCreateRoutingPolicy()
+	pdef := rp.GetOrCreatePolicyDefinition(acceptRoutePolicy)
+	stmt, _ := pdef.AppendNewStatement("20")
+	stmt.GetOrCreateActions().PolicyResult = oc.RoutingPolicy_PolicyResultType_ACCEPT_ROUTE
+	gnmi.Replace(t, dut, gnmi.OC().RoutingPolicy().PolicyDefinition(acceptRoutePolicy).Config(), pdef)
+}
+
+// configureDUTISIS configure ISIS on DUT
+func configureDUTISIS(t *testing.T, dut *ondatra.DUTDevice, aggIDs []string) {
```

Comment:
The internal/isis library should be used for isis configuration.    We should avoid cut/paste of so much ISIS configuration code.

Reply:
I am using LAG interface which support *oc.interface  to create isis but the isissession used *ondatra.port so can't use the internal/isis library.